### PR TITLE
Create journey functionality

### PR DESCRIPTION
**Pull request by maribsta**
### PR Type 🍏

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[x] API
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] Documentation content changes
[ ] Tests
[ ] Other
```

[Kanban Link](https://www.notion.so/bekks/742fa46fd71e47728aff33e5a32e125e?v=4fe4766ee0f54d348a595b3864658662&p=56be80052d7247b8a2f16d31f2c7e8d3&pm=s)

### Context 🤷‍♀️
In order to create a journey from the admin panel we have to press a the "opprett rute"-button in order to fetch an object event and set it to the database.

### What's new? 👶
Implemented a functional button that creates a new active event, and sets the rest of the events to false. After pressing the button the user is redirected to the home (landing page) and the admin user gets a confirmation that a new event is created.

### Screenshots 🖼️
Skjermbilde 2024-07-10 kl  12 38 04
image

### CODE REVIEW

File: src/lib/api/eventApi.ts [hunk 1]
```diff
@@ -62,3 +62,27 @@ export async function createOptimalRouteText(event: Event): Promise<string> {
 
     return `Vår reiseplanlegger har beregnet en optimal rute der antall etapper er ${event?.optimalStepNumber} og reisetid er ${hours} timer, ${minutes} minutter og ${seconds} sekunder.`
 }
+
+export async function createNewEvent(
```
Comment (oscarahalvorsen) [inline src/lib/api/eventApi.ts:66]: Småpirk, men lettere å finne frem dersom funksjoner heter samme i backend og frontend, så enten:
createEvent i begge eller createNewEvent i begge.

**Review by oscarahalvorsen — CHANGES_REQUESTED** [review]
Kjempekul PR! Her går det unna:)
E

File: src/lib/api/eventApi.ts (hunk 1 above)
Comment (maribsta) [reply src/lib/api/eventApi.ts]: Takk for kommentar! Fiksa

**Review by oscarahalvorsen — APPROVED** [review]
Skikkelig bra! Gøy å endelig kunne opprette nye ruter!
Og grensesnittet er veldig bra og brukervennlig

**Pull request merged**